src/python/grpcio/grpc/_common.py

```diff
@@ -94,6 +99,19 @@ def decode(b):
       return b.decode('latin1')
 
 
+def channel_args(options):
+  if options is None:
```

Comment:
As said elsewhere, let's handle `None` at the edges of our system rather than deeper down where it complicates the actual behavioral logic about which we really care.